docs/tutorials/autogluon-cloud.md

```diff
@@ -1,2 +1,119 @@
+    cloud_output_bucket="S3_BUCKET"  # S3 bucket name where intermediate artifacts will be uploaded and trained models should be saved. You need to create this bucket beforehand.
+)
+```
+
+The util function above would give you two json files describing the trust replationship and the iam policy.
+**Make sure you review those files and make necessary changes according to your use case before applying them.**
+
+## Training
+Using `autogluon.cloud` to train AutoGluon backed models are simple and not too much different from training an AutoGluon predictor directly.
+
+Currently, `autogluon.cloud` supports training/deploying `tabular`, `multimodal`, `text`, and `image` predictors. In the example below, we use `TabularCloudPredictor` for demonstration. You can substitute it with other `CloudPredictors` easily as they share the same APIs.
+
+```{.python}
+from autogluon.cloud import TabularCloudPredictor
+train_data = 'train.csv'
+predictor_init_args = {label='label'}  # init args you would pass to AG TabularPredictor
+predictor_fit_args = {train_data, time_limit=120}  # fit args you would pass to AG TabularPredictor
+cloud_predictor = TabularCloudPredictor(
+    cloud_output_path='YOUR_S3_BUCKET_PATH'
+).fit(
+    predictor_init_args,
+    predictor_fit_args,
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+To retrieve general info about the fitted `CloudPredictor`
+
+```{.python}
+cloud_predictor.info()
+```
+
+### Reattach to a Previous Training Job
+If your local connection to the training job died for some reason, i.e. lost internet connection, your training job will still be going on SageMaker, and you can reattach to the job with another `CloudPredictor` as long as you have the job name.
+
+```{.python}
+another_cloud_predictor = TabularCloudPredictor(cloud_output_path='YOUR_S3_BUCKET_PATH')
+another_cloud_predictor.attach_job(job_name="JOB_NAME")
+```
+
+To be noticed, the reattached job will no longer give live stream of the training job's log. Instead, the log will be available once the training job is finished.
+
+## Endpoint Deployment and Real-time Prediction
+If you want to deploy a predictor as a SageMaker endpoint, which can be used to do real-time inference later, it is just one line of code:
+
+```{.python}
+cloud_predictor.deploy(
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)
+```
+
+You can also attach to a deployed endpoint,
+
+```{.python}
+cloud_predictor.attach_endpoint(endpoint="ENDPOINT_NAME")
+```
+
+To do real-time prediction,
+
+```{.python}
+result = cloud_predictor.predict_real_time(
+    'test.csv',
+    instance_type="INSTANCE_TYPE"  # Checkout supported instance and pricing here: https://aws.amazon.com/sagemaker/pricing/
+)  # result will be a pandas dataframe
```

Comment:
In-case we don't already support this, should also allow for pandas dataframe, which can then be saved to csv/parquet and then uploaded akin to specifying `test.csv`. This option should be mentioned here.

Reply:
We do support. I'll mention it